lib/internal/Magento/Framework/Backup/AbstractBackup.php

```diff
@@ -17,56 +17,62 @@
      *
      * @var string
      */
-    protected $_name;
+    protected $name;
```

Comment:
Unfortunately, such kind of change is only allowed for `private` properties, see http://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/#removing-or-renaming-public-and-protected-properties

Please always create PR targeting `2.2-develop` first, ideally PRs targeting other branches should be created after it is merged.

Reply:
I will change all protected variables and functions to old state.

Reply:
👍 

Please create a new PR targeting `2.2-develop` with these changes, don't close this PR - just force push into it after `2.2` PR is processed.

Reply:
I have `2.2-develop` in local ready to push. I will take care in next PR